userguide/system.rst

```diff
@@ -1519,18 +1518,21 @@ and click |ui-add|.
    +-----------------+----------------+-------------------------------------------------------------------------------------+
    | FreeNAS URL     | string         | Hostname or IP address of the remote %brand% system. Only available                 |
    |                 |                | with *Semi-automatic* configurations. A valid URL scheme is required. Example:      |
-   |                 |                | :samp:`https://{10.231.3.76}`                                                       |
+   |                 |                | :samp:`http://{10.231.3.76}`                                                        |
    +-----------------+----------------+-------------------------------------------------------------------------------------+
-   | Username        | string         | User account name to use for logging in to the remote system                        |
+   | Auth Token      | string         | Added when *Semi-automatic* is the :guilabel:`Setup Method`. Temporary access token |
+   |                 |                | created by the remote %brand% system. To obtain this token, log in to the           |
+   |                 |                | %brand% legacy                                                                      |
```

Comment:
Do we need "legacy" here? This is the master branch anyway...

Reply:
I wrote it like this because the default in 11.3 is to show logging in to the new UI, so thought it could help prevent confusion to make the distinction. Also, that specific button in Storage/Replication Tasks is only visible in the legacy UI. If you think it's unnecessary, I can adjust it.